app/workflow_executors/step_executors/impl/next_js_server_test_executor.go

```diff
@@ -169,16 +168,16 @@ func (e NextJsServerStartTestExecutor) Execute(step steps.ServerStartTestStep) e
 }
 
 func (e NextJsServerStartTestExecutor) AnalyseBuildLogs(buildLogs, directoryPlan, apiKey string) (bool, map[string]interface{}, error) {
-	fmt.Println("Analysing Build Logs", buildLogs)
+	fmt.Println("____Analysing Build Logs____", buildLogs)
```

Comment:
lets use logger as a standard practice anyways we were moving to it initially use zap logger